plugin/mutate/source/dextool/plugin/mutate/backend/database/package.d

```diff
@@ -279,6 +279,7 @@ struct Database {
             t0.mp_id = t1.id AND
             t1.file_id = t2.id AND
             t2.path = :path
+            GROUP BY line, status
```

Comment:
How will this work for mutants that are of the same kind, are on the same line and have the same status but mutate the code in unique ways?
Wouldn't this mean that only one of those are shown?